### PR TITLE
feat: upgrade oa verify and dns-did

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: src/doc.test.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      verificationMethod: "did:ethr:0xE39479928Cc4EfFE50774488780B9f616bd4B830#controller",
+      signature:
+        "0xd6f523ad00d129529672195b3da7fd1ddaed34551e03eadc4fc524bf8c200fe441f86d59f3895e52d1e1dd1788f79af7a164b432d2948676ec3a956cfadd13ec1c",
+    },
+  ],
+};
+
+it("works", async () => {
+  const res = await verify(doc, { network: "ropsten" });
+  console.log(JSON.stringify(res, null, 2));
```
Comment (Nebulis) [inline src/doc.test.ts:100]: :eyes:

File: src/doc.test.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import {
+    openAttestationVerifiers,
+    verificationBuilder,
+    VerificationFragmentType,
+    VerificationManagerOptions,
+    Verifier,
+    CodedError,
+  } from "@govtechsg/oa-verify";
+  
+const doc: any = {
```
Comment (Nebulis) [inline src/doc.test.ts:12]: lazy to type ?

Comment (yehjxraymond) [reply src/doc.test.ts:12]: Lol stop reviewing, this is not for review yet, I’m rushing out

File: src/components/verify/verify-page.tsx [hunk 3]
```diff
@@ -111,37 +111,37 @@ export const VerifyPage: React.FunctionComponent = () => {
         const verificationFragment = await verify(rawDocument, {
```
Comment (Nebulis) [inline src/components/verify/verify-page.tsx:111]: let's add one integration test with a did-dns file

**Pull request merged**